### PR TITLE
Sayakd915/dashboard

### LINKED ISSUE #5: Dashboard Page
### Description

<b>create seperate branch named-> <yourname>-page or component name you are working on</b>
Component should be dynamic .
for now use placeHolders where necessary

### Preview
image

### PR DESCRIPTION
**Pull request by Sayakd915**
commit -m "Dashboard Page Added"

https://github.com/user-attachments/assets/aeaacc31-f409-4238-afb8-0285f591e271

fix #5

### CODE REVIEW

File: lib/app/modules/dashboard/controller/dashboard_controller.dart
Comment (SHREYANSHSK) [inline lib/app/modules/dashboard/controller/dashboard_controller.dart:1]: add dispose , init and onClose method that were automatically generated by get cli

**Review by SHREYANSHSK — CHANGES_REQUESTED** [review]
do the small change then i will merge

File: lib/app/modules/dashboard/controller/dashboard_controller.dart
Comment (Sayakd915) [reply lib/app/modules/dashboard/controller/dashboard_controller.dart:1]: I didn't remove these methods. They were already removed when I began writing the code

Comment (SHREYANSHSK) [reply lib/app/modules/dashboard/controller/dashboard_controller.dart:1]: RESOLVE CONFLICTS

Comment (Sayakd915) [reply lib/app/modules/dashboard/controller/dashboard_controller.dart:1]: Conflicts Resolved

**Pull request merged**